### PR TITLE
7-post-comment-by-article-id

### PR DESCRIPTION
**Pull request by Cwazycodes**

### CODE REVIEW

File: controllers/comments.controller.js [hunk 1]
```diff
@@ -16,4 +15,19 @@ const getCommentsByArticleId = (req, res, next) => {
...
-module.exports = { getCommentsByArticleId };
+const addCommentByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+
+
+  insertCommentByArticleId(article_id, username, body)
+    .then((comment) => {
+      res.status(201).json({ comment });
```
Comment (LangaLee) [inline controllers/comments.controller.js:26]: why are we using res.json here ?

File: models/comments.models.js [hunk 2]
```diff
@@ -24,6 +24,42 @@ const fetchCommentsByArticleId = (article_id) => {
...
+      `
+    INSERT INTO comments (article_id, author, body)
+    VALUES ($1, $2, $3)
+    RETURNING *;`,
+      [article_id, author, body]
+    )
+    .then((result) => {
+      return result.rows[0];
+    })
+    .catch((err) => {
```
Comment (LangaLee) [inline models/comments.models.js:52]: lets catch our errors in the controllers and handle them using next middleware

**Review by LangaLee — COMMENTED** [review]
Nice work 🙂, left feedback below

File: controllers/comments.controller.js (hunk 1 above)
Comment (Cwazycodes) [reply controllers/comments.controller.js:26]: code would not pass without it says the data must be parsed

**Pull request merged**